omnibus/package-scripts/updater-deb/postinst

```diff
@@ -0,0 +1,111 @@
+#!/bin/sh
+#
+# Perform necessary datadog-updater setup steps after package is installed.
+#
+# .deb: STEP 5 of 5
+
+INSTALL_DIR=/opt/datadog
+LOG_DIR=/var/log/datadog
+CONFIG_DIR=/etc/datadog-agent
+
+add_user_and_group() {
+    # Only create group and/or user if they don't already exist
+    NAME=$1
+    HOME_DIR=$2
+    getent group "$NAME" >/dev/null || (echo "Creating $NAME group" && addgroup --system "$NAME" --quiet)
+    set +e
+    id -u "$NAME" >/dev/null 2>&1
+    USER_EXISTS=$?
+    set -e
+    if [ ! $USER_EXISTS -eq 0 ]; then
+        echo "Creating $NAME user"
+        adduser --system "$NAME" --disabled-login --shell /usr/sbin/nologin --home "$HOME_DIR" --no-create-home --group --quiet
+    elif id -nG "$NAME" | grep --invert-match --word-regexp --quiet "$NAME"; then
+        # User exists but is not part of the $NAME group
+        echo "Adding $NAME user to $NAME group"
+        usermod -g "$NAME" "$NAME"
+    fi
+}
+
+enable_stable_agents() {
+    if command -v systemctl >/dev/null 2>&1; then
+        # Force systemd to ignore the sysvinit scripts. Only cosmetic, remove some irrelevant warnings during upgrade
+        SYSTEMCTL_SKIP_SYSV=true systemctl enable datadog-updater || true
+        SYSTEMCTL_SKIP_SYSV=true systemctl enable datadog-agent-process || true
+        SYSTEMCTL_SKIP_SYSV=true systemctl enable datadog-agent-sysprobe || true
+        SYSTEMCTL_SKIP_SYSV=true systemctl enable datadog-agent-trace || true
+        SYSTEMCTL_SKIP_SYSV=true systemctl enable datadog-agent-security || true
+        SYSTEMCTL_SKIP_SYSV=true systemctl enable datadog-agent || true
+        # experiment agents are not enabled as we don't systemctl enable them
+    fi
+
+
+}
+
+set -e
+case "$1" in
+    configure)
+        add_user_and_group 'dd-agent' $INSTALL_DIR/agents
+        add_user_and_group 'dd-updater' $INSTALL_DIR
+        usermod -g dd-agent dd-updater
+    ;;
+    abort-upgrade|abort-remove|abort-deconfigure)
+    ;;
+    *)
+    ;;
+esac
+#DEBHELPER#
+
+# Set the installation information if not already present;
+# This is done in posttrans for .rpm packages
+if [ ! -f "$CONFIG_DIR/install_info" ]; then
+
+    if command -v dpkg >/dev/null 2>&1 && command -v dpkg-query >/dev/null 2>&1; then
+        tool=dpkg
+        tool_version=dpkg-$(dpkg-query --showformat='${Version}' --show dpkg  | cut -d "." -f 1-3 || echo "unknown")
+    else
+        tool=unknown
+        tool_version=unknown
+    fi
+
+    install_info_content="---
+install_method:
+  tool: $tool
+  tool_version: $tool_version
+  installer_version: deb_package
```

Comment:
Don't we want to mention the install method is the updater?

Reply:
The install_info file will definitely need to change somehow. I would agree that this is good for initial testing, but we need to reflect the information here that the Agent is installed by the updater, but we also want to include the updater version and its install tool etc.

So I would tentatively accept this, but it will certainly need to be improved.

Reply:
I've added an extra tag `installer: updater`. I will dig deeper on how this is used